components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/rest/RESTAPIServiceImpl.java

```diff
@@ -434,4 +439,64 @@ private EndpointConfig getEndpointConfig(String endpointConfig) throws IOExcepti
         }
         return endpointConf;
     }
+
+    /**
+     * @see RESTAPIService#getClientCertificates(String)
+     */
+    public List<ClientCertMetadataDTO> getClientCertificates(String accessToken) {
+        Config config = GatewayCmdUtils.getConfig();
+        URL url;
+        HttpsURLConnection urlConn = null;
+        ClientCertificatesDTO certList;
+        List<ClientCertMetadataDTO> selectedCertificates = new ArrayList<>();
+        //calling token endpoint
+        publisherEp = publisherEp.endsWith("/") ? publisherEp : publisherEp + "/";
+        try {
+            String urlStr = publisherEp + "clientCertificates";
+            url = new URL(urlStr);
+            urlConn = (HttpsURLConnection) url.openConnection();
+            if (inSecure) {
+                urlConn.setHostnameVerifier((s, sslSession) -> true);
+            }
+            urlConn.setDoOutput(true);
+            urlConn.setRequestMethod(RESTServiceConstants.GET);
+            urlConn.setRequestProperty(RESTServiceConstants.AUTHORIZATION,
```

Comment:
We recently got a request from a user to change the Authorization header value. Therefore, shall we make this value configurable? Let's say you define the Authorization header as AUTH_HEADER. Then you need to read the value of the AUTH_HEADER from env values and if it already there, take that value as the Auth header value. Otherwise, use the default auth header value.